charts/velero/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 1.4.0
 description: A Helm chart for velero
 name: velero
-version: 2.12.13
+version: 3.12.13
```

Comment:
The version is better to start back from 0. For example: `3.0.0` or change major/minro version is enough `2.13.0`/`2.12.x`.

Reply:
Of course, sorry I spaced that